lib/chef/resource/build_essential.rb

```diff
@@ -61,7 +61,8 @@ class BuildEssential < Chef::Resource
           package "devel/m4"
           package "devel/gettext"
         when macos?
-          unless xcode_cli_installed?
+          update_label = xcode_cli_package
```

Comment:
```suggestion
          xcode_version = xcode_cli_package
```

This variable name is kind of strange to me. `xcode_version`, maybe?